website/docs/components/side-nav/partials/code/component-api.md

```diff
@@ -1,5 +1,130 @@
 ## Component API
 
+### SideNav::Wrapper
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="<:header>" @type="named block">
+    This is a named block where the content for the “header” area of the SideNav is rendered. The `SideNav::Header` component should be added here.
+  </C.Property>
+  <C.Property @name="<:body>" @type="named block">
+    This is a named block where the content for the “body” or main content of the SideNav is rendered. The `SideNav::List` component, if used, should be added here.
+  </C.Property>
+  <C.Property @name="<:footer>" @type="named block">
+    This is a named block where the content for the “footer” section of the SideNav is rendered.
+  </C.Property>
+</Doc::ComponentApi>
+
+### SideNav::Header
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="<:logo>" @type="named block">
+    This is a named block where the main product logo linked to your app’s home page will be rendered. The `SideNav::HomeLink` component should be added here.
+  </C.Property>
+  <C.Property @name="<:actions>" @type="named block">
+    This is a named block where the header “action” components will be rendered. Typically `Hds::Dropdown` components and/or `SideNav::IconButton` components will be added here. Special SideNav coordinated styling can be applied to dropdowns by adding the “hds-side-nav__dropdown” class name.
+  </C.Property>
+</Doc::ComponentApi>
+
+### SideNav::HomeLink
+
+!!! Info
+
+The `SideNav::HomeLink` component uses the generic `Hds::Interactive` component. For more details about how this utility component works please refer to [its documentation page](/utilities/interactive).
+
+!!!
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="icon" @type="string">
+    Used to show an icon. Any [icon](/icons/library) name is accepted. Typically you would add the icon name for your product.
+  </C.Property>
+  <C.Property @name="text" @type="string" @required="true">
+    The value of the aria-label. If no text value is defined an error will be thrown.
+  </C.Property>
+  <C.Property @name="color" @type="string" @required="string">
+    Used to specify an optional custom color provided as a [color token](/foundations/colors). If unspecified, it will use the SideNav’s default white text color.
+  </C.Property>
+</Doc::ComponentApi>
+
+### SideNav::IconButton
+
+!!! Info
+
+The `SideNav::IconButton` component uses the generic `Hds::Interactive` component. For more details about how this utility component works please refer to [its documentation page](/utilities/interactive).
+
+!!!
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="icon" @type="string">
+    Used to show an icon. Any [icon](/icons/library) name is accepted.
+  </C.Property>
+  <C.Property @name="ariaLabel" @type="string" @required="true">
+    The value of the aria-label. If no text value is defined an error will be thrown.
+  </C.Property>
```

Comment:
The HomeLink component uses `@text` to set the aria-label content. Would we want to make the argument names consistent?

Reply:
If we use `@text` in this context, one could assume is the text that will appear in the button, next to the icon (that's why I used that name)

That said, `The HomeLink component uses @text to set the aria-label content` this is not true (was an early implementation, but was removed: since we're spreading the attributes, consumers can pass an aria-attribute as they see fit.

Reply:
Thanks, I was looking at Showcase code in which I still had the old `@text` arg.

Should I update the `HomeLink` component to add a required `@ariaLabel` arg like `IconButton` uses?